assets/_sass/components/_about-table-contents.scss

```diff
@@ -0,0 +1,36 @@
+.about-table-contents {
+  & i {
+    color: $color-dust;
+    font-weight: bold;
+    position: relative;
+    top: 2px;
+    left: -2px;
+  }
+
+  ul {
+    li {
+      list-style-type: none;
+      line-height: 2;
+      &:first-of-type {
+        @extend %btn;
+        text-transform: uppercase;
+      }
+      & i {
```

Comment:
The `&` here is unnecessary because of the way the SCSS is structured. Because the `i` selector is nested under the `li` selector, it will be compiled out to `li i` automatically. The ampersand is used for the `:first-of-type` and `:before` because it would otherwise be compiled to `li :first-of-type` and `li :before`, which is not the same thing as `li:first-of-type` and `li:before`.